partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/dto/ClientDetailCreateRequest.java

```diff
@@ -27,7 +27,7 @@ public class ClientDetailCreateRequest {
     @NotNull
     @NotBlank
```

Comment:
@notNull is not required when using @notBlank

Reply:
Updated the same anusha